build-scripts/webpack.cjs

```diff
@@ -152,6 +154,15 @@ const createWebpackConfig = ({
           )
         ),
       !isProdBuild && new LogStartCompilePlugin(),
+      isProdBuild &&
```

Comment:
We should probably only do this in CI? And not in the actual build action?

Reply:
My test locally didn't add any noticeable build time, and the filtered files are only about 500 KB for the frontend.

I set it for any build because this way you could also use their CLI tool to compare locally (you would just need to manually download a set of dev stats first).